### PR TITLE
Vite JS Docs added 

### PR DESCRIPTION
**Pull request by Aksshay88**
## Related Issue
 #392
## Description

I have added the complete docs required for the usage of the vite js 

## Type of PR

- [X] Feature enhancement
- [X] Documentation update
## Screenshots / Videos (if applicable)

image
image
image

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [X] I have performed a self-review of my code.
- [X] I have read and followed the Contribution Guidelines.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)
-  [Vite JS](https://vitejs.dev/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: docs/ViteJs/config/buil-options.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
+# Build Options
```
Comment (Ajay-Dhangar) [inline docs/ViteJs/config/buil-options.md:1]: remove `#Build Options`

and write format like

```md
---
id: ___
title: ____
sidebar_label: _____
sidebar_position: ____
---

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
## Important Note:
Please work on only one file at a time to ensure ease of understanding and bug fixing. Attempting to work on multiple files simultaneously requires a deep understanding of our structure. If you are not familiar with it, focus on one file at a time. Resources have been provided for your reference.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

**Comment by Aksshay88** [comment]
@Ajay-Dhangar sure will work towards one file .Thanks for suggestiong this .I refer this one .

**Comment by Ajay-Dhangar** [comment]
@Aksshay88 Please close this issue and the associated pull request. Address each item one at a time, and work on your local system for a better understanding of the code. Ensure that the code is error and bug-free before proceeding. I want this issue and pull request to be closed.

**Pull request closed without merging**